.github/workflows/ft_tests.yml

```diff
@@ -0,0 +1,54 @@
+name: Free Threading Tests
+on: [push, pull_request]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install uv and set the python version
+        uses: astral-sh/setup-uv@v5
+        with:
+          version: "0.7.2"
+          python-version: 3.13t
+          enable-cache: false
+      - name: Set up Python
+        run: uv python install
+      - name: Install Dependencies
+        run: |
+          set -x
+          set -e
+          uv pip install packaging setuptools \
+          pyflakes pytest-cov pytest-flakes \
+          pytest-timeout pytest-durations
+          # Test dep, consider skipping
+          uv pip install sympy hypothesis
+          uv pip install --pre --upgrade -i \
+          https://pypi.anaconda.org/scientific-python-nightly-wheels/simple \
+          cython
+          uv pip install --pre --upgrade -i \
+          https://pypi.anaconda.org/scientific-python-nightly-wheels/simple \
+          numpy
+          uv pip install pytest-run-parallel
+      - name: Disable Coverage # see gh-198
+        run: |
+          sed -i '/plugins = Cython.Coverage/d' .coveragerc
+          sed -i 's/addopts/\#addopts/' pytest.ini
+      - name: In-place build
+        run: |
+          export CFLAGS="-fsanitize=address -fno-omit-frame-pointer -O0 -g"
+          export LDFLAGS="-fsanitize=address"
```

Comment:
I don’t think this would be needed either if you used a CPython build instrumented for ASAN